setup.tpl

```diff
@@ -88,4 +89,4 @@ vault write auth/azure/login \
   vm_name="`hostname`"
 EOF
 
-chmod +x /tmp/azure_auth.sh
\ No newline at end of file
+chmod +x /tmp/azure_auth.sh
```

Comment:
I never understood the `/tmp/azure_auth.sh` file... Is it even executed?

Reply:
No, it's not, it's an helper